app/src/main/java/io/homeassistant/companion/android/settings/notification/views/NotificationChannelView.kt

```diff
@@ -0,0 +1,115 @@
+package io.homeassistant.companion.android.settings.notification.views
+
+import android.app.NotificationChannel
+import android.os.Build
+import androidx.annotation.RequiresApi
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.material.Divider
+import androidx.compose.material.Icon
+import androidx.compose.material.Scaffold
+import androidx.compose.material.SnackbarResult
+import androidx.compose.material.Text
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.Delete
+import androidx.compose.material.icons.filled.Edit
+import androidx.compose.material.rememberScaffoldState
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.rememberCoroutineScope
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.unit.dp
+import io.homeassistant.companion.android.common.R
+import io.homeassistant.companion.android.common.sensors.SensorWorkerBase
+import io.homeassistant.companion.android.settings.notification.NotificationViewModel
+import io.homeassistant.companion.android.websocket.WebsocketManager
+import kotlinx.coroutines.launch
+
+@RequiresApi(Build.VERSION_CODES.O)
+@Composable
+fun NotificationChannelView(
+    notificationViewModel: NotificationViewModel
+) {
+    val scaffoldState = rememberScaffoldState()
+    val scope = rememberCoroutineScope()
+    val context = LocalContext.current
+    Scaffold(
+        scaffoldState = scaffoldState
+    ) {
+        val sortedChannelList = notificationViewModel.channelList[0].toList().sortedBy { it.name.toString() }

```

Comment:
- `toList()` is unnecessary because it is already a list
- Because you're always sorting the list, this should be done in the viewmodel instead